adder/sharding/dag.go

```diff
@@ -54,7 +45,7 @@ func CborDataToNode(raw []byte, format string) (ipld.Node, error) {
 	if err != nil {
 		return nil, err
 	}
-	shardNode, err := ipld.Decode(shardBlk)
+	shardNode, err := cbor.DecodeBlock(shardBlk)
```

Comment:
Looks like this only uses dag-cbor decoding so just went with directly decoding here